src/linker/Linker/LinkContext.cs

```diff
@@ -106,6 +107,11 @@ public class LinkContext : IDisposable
 			set { _keepTypeForwarderOnlyAssemblies = value; }
 		}
 
+		public bool SingleFileAnalysis {
```

Comment:
nit: similar methods in LinkContext tend to be named Is<Something>, maybe name this `IsSingleFile`?

Reply:
Also, why expand the getter/settter. 
```C#
public bool SingleFileAnalysis { get; set; }
```